v1.1/alter-table.md

```diff
@@ -20,3 +20,7 @@ Subcommand | Description
 [`RENAME COLUMN`](rename-column.html) | Change the names of columns.
 [`RENAME TABLE`](rename-table.html) | Change the names of tables.
 `SPLIT AT` | *(Documentation pending)* Potentially improve performance by identifying ideal locations to split data in the key-value layer.
+
+## Viewing Schema Changes
```

Comment:
Love that you adding this tip to all the relevant schema change statement docs. On this page, it might be nice to add a `Subcommands` h2 header above "For information on using.." and then a toc above that to help users know what's on the page. 

Then I'd also put the New in v1.1 callout next to this header to get that fact to show up in the toc:

```
## Viewing Schema Changes <span class="version-tag">New in v1.1</span>
```

Reply:
Done.